airflow/cli/commands/db_command.py

```diff
@@ -71,6 +71,7 @@ def shell(args):
                 database = {url.database}
                 """
             ).strip()
+            print(content)
```

Comment:
Just one thing -> can you remove that - seems like a debug statement

Reply:
Yes， my mistake. Thank you very much for your guidance. It’s the first time I participated in contributing to an open source project

Reply:
Happens to everyone. Nice one for the  first contribution. We hope to see you more frequently around!